bindings/tpython/src/TPython.cxx

```diff
@@ -352,6 +348,8 @@ void TPython::ExecScript(const char *name, int argc, const char **
    delete[] argv2;
 #else
 // TODO: fix this to work like above ...
+   (void)argc;
```

Comment:
`argc` is already used in code that will be compiled for both Py2 and Py3, so we don't need this here?

Reply:
Yes and no: `argc` is "used but not set". This is the warning that I'm trying to address here.

Reply:
I see, so with `argc += 1` it's first used (read) to be set, but we still get the warning :)

Reply:
Yes, recent Clang versions got a bit picky about those. I mean, it's correct in principle...